### PR TITLE
Create a waste-items API

### LINKED ISSUE #11: Generate API for waste items
- [x] Create unique IDs
- [x] Add small description
- [x] Set link to Waste-Pics
- [x] Set type of Waste

### PR DESCRIPTION
**Pull request by ShiningElektra**
fix #11
- Create an API with waste-items
- Add waste items images to public-folder
- Add typescript of APIWaste
- Create API Routes

### CODE REVIEW

File: server/db.json [hunk 1]
```diff
@@ -0,0 +1,212 @@
+{
+  "waste": [
+    {
+      "id": "1301",
+      "description": "dropped icecream",
+      "typoeof": "rest",
```
Comment (phlgr) [inline server/db.json]:
```suggestion
      "typeof": "rest",
```
There seems to be a typo

**Review by phlgr — CHANGES_REQUESTED** [review]
Nice job, fix the typos then you are good to go 🚀

File: server/db.json [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+      "id": "1212",
+      "description": "white plastic bag",
+      "typoeof": "recycle",
+      "image": "/recycle_12.svg",
+      "score": "1",
+      "credits": "Lebensmittel Vektor erstellt von brgfx - de.freepik.com"
+    },
+    {
+      "id": "1101",
+      "description": "old magazin",
```
Comment (fre-ben) [inline server/db.json]: Just a small typo "old magazin" -> "old magazin**e**"

File: server/db.json [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+      "score": "1",
+      "credits": "Lebensmittel Vektor erstellt von brgfx - de.freepik.com"
+    },
+    {
+      "id": "1001",
+      "description": "banana peel",
+      "typoeof": "rest",
+      "image": "/bio_01.svg",
+      "score": "1",
+      "credits": "Lebensmittel Vektor erstellt von brgfx - de.freepik.com"
```
Comment (fre-ben) [inline server/db.json]: "banana peel" item is doubled

**Review by fre-ben — CHANGES_REQUESTED** [review]
LGTM 👍

**Review by RonjaMeinecke — APPROVED** [review]
GJ🚀

**Review by m-wiemers — APPROVED** [review]
The sun rises when this code appears ☀

File: utils/api.ts [hunk 4]
```diff
@@ -0,0 +1,8 @@
+export type APIWaste = {
+  id: number;
+  description: string;
+  typeof: "rest" | "recycle" | "paper" | "bio";
```
Comment (lmachens) [inline utils/api.ts]: I would rename this to `type`. `typeof` sounds like a function or operator.

Comment (lmachens) [reply utils/api.ts]: Maybe rename `rest` to `residual`

File: utils/api.ts [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+  image: string;
```
Comment (lmachens) [inline utils/api.ts]: `image` could be a base64 encoded string but you like to set the path/src of it. Maybe rename to `imageSrc`?

File: server/db.json [hunk 6]
```diff
@@ -0,0 +1,212 @@
...
+    {
+      "id": "1003",
+      "description": "lettuce leaf leftover",
+      "typoeof": "rest",
+      "image": "/bio_03.svg",
+      "score": "1",
+      "credits": "Lebensmittel Vektor erstellt von brgfx - de.freepik.com"
+    },
+    {
+      "id": "1004",
```
Comment (lmachens) [inline server/db.json]: `APIWaste` says that `id` is a number

File: server/db.json [hunk 7]
```diff
@@ -0,0 +1,212 @@
...
+      "description": "rotten peppers",
+      "typoeof": "rest",
+      "image": "/bio_04.svg",
+      "score": "1",
```
Comment (lmachens) [inline server/db.json]: `APIWaste` says that `score` is a number`

**Review by lmachens — CHANGES_REQUESTED** [review]
The file size of the images is huge. I think the export of Illustrator is the reason. 
I did a small test and was able to reduce bio_02.svg (218kb) to bio_02.png (4kb).

I would love to see an API route to test it and use the typing.
Maybe `/api/waste-items`?

This pull request is not assigned to your project board.

File: server/db.json [hunk 8]
```diff
@@ -0,0 +1,204 @@
+{
+  "waste-items": [
+    {
+      "id": 1301,
+      "description": "dropped icecream",
+      "type": "residual",
+      "imageSrc": "/residual_01.svg",
```
Comment (lmachens) [inline server/db.json]: You converted your images in `.png`. 
```suggestion
      "imageSrc": "/residual_01.png",
```

**Review by lmachens — CHANGES_REQUESTED** [review]
Good changes :)

File: server/db.json [hunk 9]
```diff
@@ -0,0 +1,204 @@
...
+    {
+      "id": 1003,
+      "description": "lettuce leaf leftover",
+      "type": "residual",
+      "imageSrc": "/bio_03.svg",
+      "score": 1,
+      "credits": "Lebensmittel Vektor erstellt von brgfx - de.freepik.com"
+    },
+    {
+      "id": 1004,
```
Comment (RonjaMeinecke) [inline server/db.json:196]: peppers, lettuce, eggshell, banana peel, fries, pizza, bone... and so on is biodegradable and therefore should be the type "bio"

**Review by RonjaMeinecke — APPROVED** [review]
🚀well done

**Review by phlgr — APPROVED** [review]
🚀

File: server/db.json (hunk 9 above)
Comment (ShiningElektra) [reply server/db.json:196]: thx, this will change in level 2 :-)

**Pull request merged**